Expose `Comparison` class

safo6m opened the pull request:
`Comparison` is mentioned in the `Readme` as an argument to some operators but is not exposed to the developer to create instances of the `Comparison` class.

RomiC commented:
In order not to forget to export it in the future, could you pls add the corresponding assertion to [this scenario](https://github.com/RomiC/rsql-builder/blob/master/tests/index.spec.ts#L4)?

safo6m commented:
Thanks for the review. I've changed it as requested.

RomiC commented:
@safo6m sorry, I've just found and fixed one build issue (#288) I have for a while. Could you pls rebase your changes on the latest master branch, so I can publish a new version with your changes?

safo6m commented:
> @safo6m sorry, I've just found and fixed one build issue (#288) I have for a while. Could you pls rebase your changes on the latest master branch, so I can publish a new version with your changes?

As you already merged your fix to `master`, I suppose the rebase is not necessary as you will probably make a new release from `master`. Regardless, I've updated my branch with the latest `master` so you should have a clean slate.
